lib/mix/tasks/compile.nerves_package.ex

```diff
@@ -41,31 +40,50 @@ defmodule Mix.Tasks.Compile.NervesPackage do
     end
   end
 
-  @doc false
-  defp bootstrap_check(true), do: :ok
+  defp bootstrap_check!() do
+    cond do
+      bootstrap_started?() ->
+        :ok
 
-  defp bootstrap_check(false) do
-    error =
-      """
-      Compiling Nerves packages requires nerves_bootstrap to be started, which ought to
-      happen in your generated `config.exs`. Please ensure that MIX_TARGET is set in your environment.
-      """ <>
-        if in_umbrella?(File.cwd!()) do
+      not bootstrap_installed?() ->
+        Mix.raise("""
+        Compiling Nerves packages requires the nerves_bootstrap archive which is missing
+        from the Elixir version currently in use (#{System.version()}).
```

Comment:
@JasonTrue I altered the error message and added the Elixir version based on our Slack conv